packages/sanes-chrome-extension/src/routes/signup/test/operateSignup.ts

```diff
@@ -82,13 +98,13 @@ export const submitNewAccount = async (
   accountName: string,
   password: string,
 ): Promise<void> => {
-  const inputs = TestUtils.scryRenderedDOMComponentsWithTag(newAccountDom, 'input');
-  expect(inputs.length).toBe(3);
```

Comment:
This check is not required because we are checking amount of fields in another test.